[WOOMOB-3438] Fix bulk regular price update for variable products (#17556)
woocommerce/woocommerce-ios: RELEASE-NOTES.txt

25.3
-----

> - [*] Products: Fixed bulk Regular Price update failing for variable products whose variations share a duplicate SKU or GTIN. [https://github.com/woocommerce/woocommerce-ios/pull/17556]

Reviewer: This reads a bit technical to me. To make it clearer which problem this fixes for the user, what do you think about something like this?

```suggestion
- [*] Products: Fixed bulk Regular Price update failing with "Unable to update price" on variable products. [https://github.com/woocommerce/woocommerce-ios/pull/17556]
```
